app/controllers/users_controller.rb

```diff
@@ -1,3 +1,26 @@
 class UsersController < ApplicationController
-  def new; end
+  def show
+    @user = User.find(params[:id])
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      flash[:success] = "Welcome to the Sample App!"
+      redirect_to @user
+    else
+      render "new"
```

Comment:
em chuyển qua dùng symbol thay cho string nhé
```suggestion
      render :new
```